### PR TITLE
chore: PactNet v4 announcement

### PR DESCRIPTION
**Pull request by adamrodger**

### CODE REVIEW

**Review by ajpatterson7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,17 +1,43 @@
...
+- The Rust implementation is receiving all the new features and fixes, including Pact specification v4 support in future
+- A number of long-standing issues will be resolved, such as:
+  - Filepath lengths on Windows will no longer be an issue since the Rust implementation is a single file under the `bin/` folder
+  - There is no child Ruby process still running if you abort an interactive debugger session, since the Rust implementation runs in-proc
+  - The mock server can allocate an unused port dynamically, so you no longer need to provide one that you know is unused
+  - You no longer need to remember to call `ClearInteractions()` between each consumer test
+
+We're working really hard to get to a beta release, but until then you can check out our progress on the `feature/4.0.0` branch.
+
+We understand that breaking changes like these can be difficult for users of PactNet, but the major shift in the reference implementation
```
Comment (mefellows) [inline README.md:21]: @ajpatterson7 A link to a migration guide here would be 👌

File: README.md [hunk 2]
```diff
@@ -33,8 +59,8 @@ Massive thanks to the SEEK team for all the time and hard work put into this lib
 1. When debugging a test locally (either consumer or provider) if you click the stop button in your test runner, it will abort the process abruptly and the ruby runtime will not get cleaned up. If you do this, simply kill the ruby process from your task/process manager. We recommend you play the test through to the end to avoid this issue. See https://github.com/pact-foundation/pact-net/issues/108 for more details.
 
 ## Usage
-Below are some samples of usage.  
-For examples of Version 2 usage, please see the [Samples](https://github.com/pact-foundation/pact-net/blob/master/Samples/EventApi/Consumer.Tests/EventsApiConsumerTests.cs).  
+Below are some samples of usage.
```
Comment (mefellows) [inline README.md:62]: Should we link to https://github.com/DiUS/pact-workshop-dotnet-core-v3/?

**Review by mefellows — APPROVED** [review]
Mostly comments, but no reason not to merge and address later if you see fit.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,43 @@
...
+
+- **PACT SPECIFICATION V3 SUPPORT INCLUDING MESSAGE PACTS**
+- A new fluent API which guides developers into the pit of success more often
+- There is now one NuGet to install which handles all supported platforms instead of one per OS/arch combination
+- The Rust implementation is receiving all the new features and fixes, including Pact specification v4 support in future
+- A number of long-standing issues will be resolved, such as:
+  - Filepath lengths on Windows will no longer be an issue since the Rust implementation is a single file under the `bin/` folder
+  - There is no child Ruby process still running if you abort an interactive debugger session, since the Rust implementation runs in-proc
+  - The mock server can allocate an unused port dynamically, so you no longer need to provide one that you know is unused
+  - You no longer need to remember to call `ClearInteractions()` between each consumer test
```
[the remark this answers is not in the document]
Comment (adamrodger) [reply README.md:17]: Unfortunately not :( On Windows and Mac, at least, each consumer test takes >2s whereas on the Ruby version this isn't the case. It's much faster on Linux though (check the CI runs for 4.x to see). I've talked about that previously on Slack.

**Pull request merged**